openpype/hosts/houdini/plugins/create/create_karma_rop.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin to create Karma ROP."""
+from openpype.hosts.houdini.api import plugin
+from openpype.pipeline import CreatedInstance
+from openpype.lib import BoolDef, EnumDef, NumberDef
+
+
+class CreateKarmaROP(plugin.HoudiniCreator):
+    """Karma ROP"""
+    identifier = "io.openpype.creators.houdini.karma_rop"
+    label = "Karma ROP"
+    family = "karma_rop"
+    icon = "magic"
+    defaults = ["master"]
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        import hou  # noqa
+
+        instance_data.pop("active", None)
+        instance_data.update({"node_type": "karma"})
+        # Add chunk size attribute
+        instance_data["chunkSize"] = 10
+        # Submit for job publishing
+        instance_data["farm"] = True
+
+        instance = super(CreateKarmaROP, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)  # type: CreatedInstance
+
+        instance_node = hou.node(instance.get("instance_node"))
+
+        ext = pre_create_data.get("image_format")
+
+        filepath = "{}{}".format(
+            hou.text.expandString("$HIP/pyblish/render/"),
+            "{}/{}.$F4.{}".format(subset_name, subset_name, ext)
+        )
+        checkpoint = "{}{}".format(
+            hou.text.expandString("$HIP/pyblish/"),
+            "{}.$F4.checkpoint".format(subset_name)
+        )
+
+        usd_directory = "{}{}".format(
+            hou.text.expandString("$HIP/pyblish/renders/usd_renders/"),
+            "{}_$RENDERID".format(subset_name, subset_name, ext)
+        )
+
+        parms = {
+            # Render Frame Range
+            "trange": 1,
+            # Karma ROP Setting
+            "picture": filepath,
+            # Karma Checkpoint Setting
+            "productName": checkpoint,
+            # USD Output Directory
+            "savetodirectory": usd_directory,
+        }
+
+        res_x = pre_create_data.get("res_x")
+        res_y = pre_create_data.get("res_y")
+
+        if self.selected_nodes:
+            # If camera found in selection
+            # we will use as render camera
+            camera = None
+            for node in self.selected_nodes:
+                if node.type().name() == "cam":
+                    camera = node.path()
+                    camera_node = hou.node(camera)
```

Comment:
If I am not mistake it should be the same as `node` in line 67, right?
`camera_node = node`